packages/cli/src/api/pseudoLocalize.test.ts

```diff
@@ -92,4 +92,14 @@ describe("PseudoLocalization", () => {
     expect(pseudoLocalize("replace {count}")).toEqual("ŕēƥĺàćē {count}")
     expect(pseudoLocalize("replace { count }")).toEqual("ŕēƥĺàćē { count }")
   })
+
+  it("multiple plurals is wrong45", () => {
```

Comment:
this is the test that shows why the currently implemented ICU support in ` pseudoLocalize.ts` is wrong